Wikipedia/Code/WMFTableViewCellWithCustomMinusButton.m

```diff
@@ -0,0 +1,71 @@
+#import "WMFTableViewCellWithCustomMinusButton.h"
+#import <Masonry/Masonry.h>
+
+@interface WMFTableViewCellWithCustomMinusButton ()
+
+@property (strong, nonatomic) UIButton* minusButton;
+@property (strong, nonatomic) CALayer* cellWhiteLayer;
+@property (strong, nonatomic) CALayer* minusButtonWhiteLayer;
+
+@end
+
+@implementation WMFTableViewCellWithCustomMinusButton
+
+- (void)layoutSubviews {
+    [super layoutSubviews];
+    
+    // HAX: ensure the custom button is always on top.
+    [self.minusButton.superview bringSubviewToFront:self.minusButton];
```

Comment:
I don't think it triggers a layout pass, but doesn't hurt to check it I guess. Fixed.